helm/memcached-operator/README.md

```diff
@@ -0,0 +1,103 @@
+# Memcached Helm Operator
+
+## Overview
+
+This Memcached operator is a simple example operator based in Helm built with the [Operator SDK][operator_sdk].
+
+## Prerequisites
+
+- [docker][docker_tool] version 17.03+
+- [kubectl][kubectl_tool] v1.11.3+
+- [operator_sdk][operator_install]
+- Access to a Kubernetes v1.11.3+ cluster
+
+## Getting Started
+
+### Cloning the repository
+
+Checkout this Memcached Operator repository
+
+```
+$ mkdir operator-framework
+$ cd operator-framework
+$ git clone https://github.com/operator-framework/operator-sdk-samples.git
+$ cd operator-sdk-samples/helm/memcached-operator
+```
+
+### Building the operator
+
+Build the Memcached operator image and push it to a public registry, such as quay.io:
+
+```
+$ export IMAGE=quay.io/example-inc/memcached-operator:v0.0.1
+$ operator-sdk build $IMAGE
+$ docker push $IMAGE
+```
+
+**NOTE** The `quay.io/example-inc/memcached-operator` is an example. You should build and push the image for your repository. 
+
+### Using the image
+
+```
+# Update the operator manifest to use the built image name (if you are performing these steps on OSX, see note below)
+$ sed -i 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
+# On OSX use:
+$ sed -i "" 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
+```
```

Comment:
The formatting here isn't great. I'd suggest this:

`````
Update the operator manifest to use the built image name (if you are performing these steps on OSX, see note below)
```sh
$ sed -i 's|REPLACE_IMAGE|quay.io/example/memcached-operator|g' deploy/operator.yaml
```

**Note**
If you are performing these steps on OSX, use the following `sed` command instead:
```sh
$ sed -i "" 's|REPLACE_IMAGE|quay.io/example/memcached-operator|g' deploy/operator.yaml
```
`````